cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -12,10 +12,19 @@ const tests = ({ service }) =>
 
     describe('ATI', () => {
       it('should have a noscript tag with an 1px image with the ati url', () => {
-        cy.hasNoscriptImgAtiUrl(
-          envConfig.atiUrl,
-          config[service].isWorldService ? envConfig.atiAnalyticsWSBucket : '',
-        );
+        if (service !== 'japanese') {
```

Comment:
It'd be better if you flipped the if statement here. e.g. `if (service === 'japanese') {...}` to make it explicit.

Reply:
personally ide prefer using `if (isGNL(service)) { ` everywhere, makes it clear why we randomly exclude japanese for people not familiar with all the BBC product confusion 🙈